model/docs/example-with-datasets.md

```diff
@@ -339,7 +339,7 @@ hosp_model.run(
     num_samples=2000,
     num_warmup=2000,
     n_timepoints=dat2.shape[0] - 1,
-    observed_hospitalizations=dat2,
+    observed_admissions=dat2,
```

Comment:
should probably rename `dat2` to something more specific, but that can be a separate PR